src/client_backend/triton/CMakeLists.txt

```diff
@@ -30,10 +30,6 @@ if(${TRITON_ENABLE_ZLIB})
   find_package(ZLIB REQUIRED)
 endif()
 
-if(${TRITON_ENABLE_GPU})
-  find_package(CUDAToolkit REQUIRED)
-endif()
-
```

Comment:
None of the files in this directory/library include `<cuda_runtime_api.h>` anymore cuz those types are all encapsulated in `cuda_runtime_library_manager.h`.